web/content/docs/devguide/development-workflows/branching-model.pandoc

```diff
@@ -0,0 +1,36 @@
++++
+date = "2018-02-23T15:28:13+01:00"
+title = "Branching model"
+author = "Lars Bilke"
+weight = 1011
+
+[menu]
+  [menu.devguide]
+    parent = "development-workflows"
++++
+
+## Forking workflow
+
+Git is very flexible in organizing a distributed development team. We use a so called **Forking workflow**.
+
+The following explanation is taken from an [in-depth article](https://www.atlassian.com/git/workflows#!workflow-forking) on that model:
+
+> Instead of using a single server-side repository to act as the *central* codebase, it gives every developer a server-side repository. This means that each contributor has not one, but two Git repositories: a private local one and a public server-side one.
+>
+> The main advantage of the Forking Workflow is that contributions can be integrated without the need for everybody to push to a single central repository. Developers push to their own server-side repositories, and only the project maintainer can push to the official repository. This allows the maintainer to accept commits from any developer without giving them write access to the official codebase.
+>
+> The result is a distributed workflow that provides a flexible way for large, organic teams (including untrusted third-parties) to collaborate securely. This also makes it an ideal workflow for open source projects.
+> <cite> https://www.atlassian.com/git/workflows#!workflow-forking </cite>
+>
+
+The workflow is summarized in the following image from the [GitHub blog](https://github.com/blog/2042-git-2-5-including-multiple-worktrees-and-triangular-workflows):
+
+![](https://cloud.githubusercontent.com/assets/1319791/8943755/5dcdcae4-354a-11e5-9f82-915914fad4f7.png)
```

Comment:
Just a notice; This image is not shown in the jenkins' artifacts, although it is a valid url pointing to right image.

Reply:
Yes this is security feature of Jenkins. Only white-listed things can be fetched from remote locations. FYI the white-list is defined [here](https://gitlab.opengeosys.org/bilke/ogs-ansible/blob/master/roles/jenkins/defaults/main.yml#L4).